mvpa2/algorithms/searchlight_hyperalignment.py

```diff
@@ -332,6 +336,8 @@ def __init__(self, **kwargs):
         self.ndatasets = 0
         self.nfeatures = 0
         self.projections = None
+        # This option makes the roi_seed in each SL to be selected during feature selection
+        self.force_roi_seed = True
```

Comment:
before next release we should hide all those above as protected (`_force_roi_seed`) or private (`__force_roi_seed`) -- they shouldn't be public modifiable attributes.